index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined">
+    <link rel="stylesheet" href="style.css">
+    <title>Alpha portifolio</title>
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo">Alpha Portifolio</a>
+        <nav><i class="fa fa-bars"></i></nav>
+    </header>
+    <main>
+        <section class="hdln-section">
+            <h1 class="name-intro">
+                I’m Alpha <br>
+                Glad to see <br> you!
+            </h1>
+            <h5 class="personal-info">
+                I’m a software developer! I can help you build a product , feature or 
+                website Look through some of my work and experience! If you like 
+                what you see and have a project you need coded, don’t hestiate to contact me.
+            </h5>
```

Comment:
**Optional:**
- [ ] You should use the `p` tag here instead of `h1` as the previous reviewer asked in her last comment.

Reply:
The previous reviewer requested for use of hx at the beginning. The last request was using p tag. I alerted her and she has replied now saying that the hx tag had been removed and I had sent the pull request much earlier before her reply. Hope it doesn't affect my submission chances. Thank you. Waiting for your reply before making the last changes.